playground/nextjs/src/pages/notifications/index.tsx

```diff
@@ -1,16 +1,53 @@
 import Title from '@/components/Title';
 import { novuConfig } from '@/utils/config';
 import { Inbox, Notifications } from '@novu/react';
+import { useState } from 'react';
 
 export default function Home() {
+  const [count, setCount] = useState(0);
```

Comment:
This is the playground code to verify that the state changes are propagated but the `Notifications` component doesn't get remounted.